testnets/holesky/QUICK_START.md

```diff
@@ -202,26 +202,28 @@ Then you can follow these steps to register your operator:
    The wETH vault is deployed at: `0xC56Ba584929c6f381744fA2d7a028fA927817f2b`
 
    ```bash
-   python3 symb.py opt-in-vault <vault_address> \
-       --private-key <operator_private_key> \
-       --chain holesky \
-       --provider http://localhost:8545
+   python3 symb.py --chain holesky \
+       --provider http://localhost:8545 \
+       opt-in-vault <vault_address> \
+       --private-key <operator_private_key>
    ```
 
 4. Deposit collateral into the vault you opted-in to with
    [deposit](https://docs.symbiotic.fi/guides/cli/#deposit):
 
    Here you can re-use the same vault address as above. The amount should be in ETH (e.g. '1' for 1 ETH).
    You MUST set the `on_behalf_of` address (i.e. the third argument in the below command) to your **operator** address.
+   - NOTE: You need to hold the respective token that the vault accepts in your operator wallet (wstETH rETH stETH wETH cbETH mETH).
+     Otherwise you will receive an error (Failed! Reason: 0x1425ea42).
 
    ```bash
-   python3 symb.py deposit <vault_address> <amount> <operator_address> \
-       --private-key <operator_private_key> \
-       --chain holesky \
-       --provider http://localhost:8545
+   python3 symb.py --chain holesky \
+       --provider http://localhost:8545 \
+       deposit <vault_address> <amount> <operator_address> \
+       --private-key <operator_private_key>
    ```
 
-5. Finally register into the BoltManager contract with `bolt` CLI:
+6. Finally register into the BoltManager contract with `bolt` CLI:
```

Comment:
I can't find step 5, so I think this change and the one below should be reverted!

Reply:
Done in followup PR

Reply:
https://github.com/chainbound/bolt/pull/720